themes/default/content/blog/pulumi-go-boilerplate-v2/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "The Easier Way to Create Pulumi Providers in Go"
+authors: ["kyle-dixler"]
+tags: ["go", "packages"]
+meta_desc: "We are thrilled to announce a major update to the Pulumi Provider Boilerplate that simplifies the provider development process."
+date: "2023-01-03"
+---
+
+## Background
+
+The Pulumi Provider Boilerplate [(view on GitHub)](https://github.com/pulumi/pulumi-provider-boilerplate) is a template repository
```

Comment:
Does the README need to be updated on this repo to align with the changes announced here?